src/gc.h

```diff
@@ -233,14 +233,14 @@ STATIC_INLINE jl_gc_pagemeta_t *pop_lf_page_metadata_back(jl_gc_global_page_pool
 #define REGION0_PG_COUNT (1 << 16)
 #define REGION1_PG_COUNT (1 << 16)
 #define REGION2_PG_COUNT (1 << 18)
-#define REGION0_INDEX(p) (((uintptr_t)(p) >> 14) & 0xFFFF) // shift by GC_PAGE_LG2
+#define REGION0_INDEX(p) (((uintptr_t)(p) >> GC_PAGE_LG2) & 0xFFFF) // shift by GC_PAGE_LG2
 #define REGION1_INDEX(p) (((uintptr_t)(p) >> 30) & 0xFFFF)
 #define REGION_INDEX(p)  (((uintptr_t)(p) >> 46) & 0x3FFFF)
 #else
 #define REGION0_PG_COUNT (1 << 8)
 #define REGION1_PG_COUNT (1 << 10)
 #define REGION2_PG_COUNT (1 << 0)
-#define REGION0_INDEX(p) (((uintptr_t)(p) >> 14) & 0xFF) // shift by GC_PAGE_LG2
+#define REGION0_INDEX(p) (((uintptr_t)(p) >> GC_PAGE_LG2) & 0xFF) // shift by GC_PAGE_LG2
 #define REGION1_INDEX(p) (((uintptr_t)(p) >> 22) & 0x3FF)
 #define REGION_INDEX(p)  (0)
```

Comment:
You should probably recompute all of these numbers

Reply:
Do you know how these were calculated initially? To keep the proportions correct?

Reply:
no, I don't have a reference for them, but you should make all the chunks the same size then match up all of the constants to them